docs/bnb_smart_chain/validator/index.md

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
'bnb_smart_chain/validator/security.md' contains a link 'fullnode.md'

Reply:
Yes, but the fullnode runbook is not in this scope, should fix it after adding the fullnode part.

Reply:
fixed